src/main/java/com/napier/sem/UserPrompt.java

```diff
@@ -0,0 +1,319 @@
+package com.napier.sem;
+
+import java.util.*;
+
+public class UserPrompt
```

Comment:
this is a shit name, but i can't come up with anything better really. suggestions welcome. if you see anything that has an unclear name feel free to shout it out.